superset-frontend/src/SqlLab/components/SqlEditor/index.tsx

```diff
@@ -566,6 +584,7 @@ const SqlEditor = ({
           <AntdSwitch
             checked={autocompleteEnabled}
             onChange={handleToggleAutocompleteEnabled}
+            // @ts-ignore
```

Comment:
This property does not exist. I think you can safely remove it as it's not being referenced anywhere. I searched for "autocomplete-switch" and I wasn't able to find anything.